### PR TITLE
MethodArgumentSpaceFixer - new test case

### PR DESCRIPTION
**Pull request by keradus**
closes: #3067

### CODE REVIEW

**Comment by keradus** [comment]
@nullref could you please confirm the fix ?

**Comment by nullref** [comment]
I tested it with my minimal test case and the original code I generated it from, both were processed without generating a syntax error.

File: tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php [hunk 1]
```diff
@@ -315,6 +315,18 @@ functionCall(
             ,
                 ['ensure_fully_multiline' => true],
             ],
+            'test wrongly formatted half-multiline function becomes fully-multiline' => [
+                '<?php
+f(
+    1,
+    2,
+3
```
Comment (julienfalque) [inline tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php]: Shouldn't this line be indented as well?

Comment (keradus) [reply tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php]: this rule is not changing indentation for arguments that are already on separated line, see `test ungodly code` case.
if we want to add it, it's ouf ot the scope of this fix

Comment (julienfalque) [reply tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php]: Damn it, I always forget that for this fixer! :D

**Comment by SpacePossum** [comment]
thanks @keradus :+1:

**Pull request closed without merging**